tox.ini

```diff
@@ -5,11 +5,13 @@ envlist = py36
 [testenv]
 whitelist_externals=
     rm
+    make
 
 commands=
     ; Force packaging even if setup.{py,cfg} haven't changed
     rm -rf ./kytos.egg-info/
     python setup.py ci
+    make clean
```

Comment:
Which problem is being solved by this?

Reply:
Running tox locally are lefting the directory "docs/developer/kytos_api/" behind. That command remove it.